crates/flux-refineck/src/ghost_statements/points_to.rs

```diff
@@ -250,10 +251,15 @@ struct CollectPointerToBorrows<'a> {
     tracked_places: FxHashMap<PlaceIndex, flux_rustc_bridge::mir::Place>,
     stmts: &'a mut GhostStatements,
     before_state: Vec<(PlaceIndex, FlatSet<Loc>)>,
+    results: IndexVec<BasicBlock, State>,
```

Comment:
I believe we don't need to mutate it, so we can make this slice and avoid the clone.

```suggestion
    results: &'a IndexSlice<BasicBlock, State>,
```